db/migrate/20240710210913_remove_not_null_constraint_from_staff_department.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveNotNullConstraintFromStaffDepartment < ActiveRecord::Migration[7.1]
```

Comment:
There is a data issue in the airtable staff list: not all staff members have their department listed.  Removing the `NOT NULL` constraint allows us to still import the records, even if the airtable has data missing.